src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -154,6 +139,37 @@ class TizenControls extends Controls {
         this.keydownCallback = (evt) => this.handleKeydown(evt);
         document.addEventListener('keydown', this.keydownCallback);
 
+        // To enable features like the ad skip button
+        super.enable.call(this, api, model);
+
+        // Next Up Tooltip
+        const baseControlbar = this.controlbar;
+        if (baseControlbar) {
+            const nextUpToolTip = baseControlbar.nextUpToolTip;
+            nextUpToolTip.off('all');
```

Comment:
Good catch on the nextUpTooltip check. Before enable is called, `this.controlbar` is initialized as null, so while it is defined it could be "null", so I need that check.